docs/make.jl

```diff
@@ -612,6 +618,30 @@ Documenter.DocMeta.setdocmeta!(
     recursive = true,
 )
 
+Documenter.DocMeta.setdocmeta!(
+    MathOptInterface,
+    :DocTestSetup,
+    :(import MathOptInterface as MOI);
+    recursive = true,
+)
+
+# Remove once MOI v1.41.0 is released
+MOI.Bridges.runtests(
+    MOI.Bridges.Constraint.GreaterToLessBridge,
```

Comment:
I'm wondering what this is doing here :laughing:

Reply:
Oh I thought I replied to this already: it's to work-around this bug https://github.com/jump-dev/MathOptInterface.jl/pull/2748